src/Slider.jsx

```diff
@@ -559,50 +602,40 @@ class Rheostat extends React.Component {
 
     if (this.canMove(idx, proposedPercentage)) {
       this.slideTo(idx, proposedPercentage, () => this.fireChangeEvent());
-      const { onKeyPress } = this.props;
       if (onKeyPress) onKeyPress();
     }
 
     killEvent(ev);
   }
 
-  // Apply user adjustments to position
-  userAdjustPosition(idx, proposedPosition) {
```

Comment:
Is this method used somewhere?

Reply:
`userAdjustPosition` was taken out in the internal version - I do not believe this logic is necessary?